$OpenBSD: patch-tests_CMakeLists_txt,v 1.3 2012/10/10 10:51:02 dcoppa Exp $

Add missing CppUnit include directive
(upstream git commit 72f9a96cceba0e22a672760961de3a87e291a602)

--- tests/CMakeLists.txt.orig	Thu Sep  6 20:03:15 2012
+++ tests/CMakeLists.txt	Wed Oct 10 12:23:55 2012
@@ -62,6 +62,8 @@ SET(test_runner_SRCS
   test_mpc.cpp
 )
 
+INCLUDE_DIRECTORIES(${CPPUNIT_INCLUDE_DIR})
+
 ADD_EXECUTABLE(test_runner ${test_runner_SRCS})
 TARGET_LINK_LIBRARIES(test_runner tag ${CPPUNIT_LIBRARIES})
 
